src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.csproj

```diff
@@ -259,6 +268,10 @@
     <Reference Include="mscorlib" />
     <Reference Include="System.Core" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.3' OR '$(TargetGroup)' == 'netstandard1.4'">
+    <Reference Include="System.IO" />
+    <Reference Include="System.Runtime.Handles" />
+  </ItemGroup>
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
Nit: not all of these dependencies are required for the PNSE builds but maybe it's more effort to filter them.